.config/nextest.toml

```diff
@@ -0,0 +1,4 @@
+nextest-version = "0.9.55"
```

Comment:
Q, should this stay in sync with the version used in `build-and-test.sh`? Nextest now lets you set both required and recommended versions, maybe this should match the recommended version? https://nexte.st/book/minimum-versions.html

Reply:
How would you enforce the sync?

Reply:
Maybe the run script could actually pull the version from here?

Reply:
Just via a comment is what I was thinking.

Reply:
Put in a comment re this.